### ISSUE TITLE
PEP8Bear whitespace settings support using SpaceConsistencyBear config names

### ISSUE THREAD

**Issue by jayvdb** [coala/coala-bears#844]
The PEP8 (pycodestyle) currently only has `ignore` and `select`. While that is helpful, it isnt user friendly.   It would be nice if the user could use the SpaceConsistencyBear config names.

Disabling `use_spaces` should add the tab-related pycodestyle codes to the ignore list.  e.g. `E101`. (there are a few others, IIRC)

The error code for trailing whitespace is `W291`.  It should be automatically added to the ignore list if `allow_trailing_whitespace` is enabled.

The error code for missing EOL at EOF is `W292`.  It should be automatically added to the ignore list if `enforce_newline_at_EOF` is enabled.

**Comment by sils1297** [comment]
Very important indeed.

CC @Asnelchristian

**Comment by jayvdb** [comment]
No doubt other linter bears also dont respect the SpaceConsistencyBear
settings.

**Comment by sils1297** [comment]
Yes indeed, that's why I CCed @Asnelchristian , he has been working a lot
on unifying setting names and meanings across bears.
